docs/sources/api/_index.md

```diff
@@ -209,37 +217,37 @@ $ curl -G -s  "http://localhost:3100/loki/api/v1/query" --data-urlencode 'query=
   }
 }
 ```
+Set the `x-scope-orgid` header,
+as defined in [Grafana Loki Multi-Tenancy](../operations/multi-tenancy/),
+to query specific tenants.
+Here is the same example query applied to the single tenant called `Tenant1`:
 
 ```bash
-$ curl -G -s  "http://localhost:3100/loki/api/v1/query" --data-urlencode 'query={job="varlogs"}' | jq
-{
-  "status": "success",
-  "data": {
-    "resultType": "streams",
-    "result": [
-      {
-        "stream": {
-          "filename": "/var/log/myproject.log",
-          "job": "varlogs",
-          "level": "info"
-        },
-        "values": [
-          [
-            "1568234281726420425",
-            "foo"
-          ],
-          [
-            "1568234269716526880",
-            "bar"
-          ]
-        ],
-      }
-    ],
-    "stats": {
-      ...
-    }
-  }
-}
+curl -H 'x-scope-orgid:Tenant1' \
+  -G -s "http://localhost:3100/loki/api/v1/query" \
+  --data-urlencode \
+  'query=sum(rate({job="varlogs"}[10m])) by (level)' | jq
+```
+
+To query against the three tenants `Tenant1`, `Tenant2`, and `Tenant3`,
+specify the tenant names separated by the pipe (`|`) character:
+
+```bash
+curl -H 'x-scope-orgid:Tenant1|Tenant2|Tenant3' \
+  -G -s "http://localhost:3100/loki/api/v1/query" \
+  --data-urlencode \
+  'query=sum(rate({job="varlogs"}[10m])) by (level)' | jq
+```
+The same Grafana Enterprise Logs example query against the three tenants
```

Comment:
```suggestion
The same Grafana Enterprise Logs example query against the three tenants requires the use of Basic Authentication.  
```

Reply:
This documentation is for Loki OSS right? Do we reference GEL there? I don't know the process well enough.

Reply:
Yes, @jeschkies. In the short term, we're including a GEL example in the Loki docs, so folks know that it is different for GEL. Our GEL readers have to find their info on this page.